Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -8,6 +8,143 @@
+          {
+            "id": "791b0ca8-eac5-4073-8a9a-330fbce1efb2",
+            "version": "KqlParameterItem/1.0",
+            "name": "param_check_provider_error",
+            "type": 1,
+            "query": "print allProviders=parse_json(tostring(\"{param_providers_list:escapejson}\")).value\r\n| mv-expand allProviders\r\n| extend type=parse_json(allProviders.properties).type\r\n| extend state=parse_json(allProviders.properties).provisioningState\r\n| where type==\"SapNetweaver\"\r\n| where state!=\"Failed\"\r\n| summarize iff(count() > 0,count(), -1)",
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "conditionalVisibility": {
+        "parameterName": "dummy_neverset",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "overview_section_params"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 1,
+            "content": {
+              "json": "Please add the SAP NetWeaver provider to view this workbook.",
+              "style": "warning"
+            },
+            "conditionalVisibility": {
+              "parameterName": "param_check_error",
+              "comparison": "isEqualTo",
+              "value": "1"
+            },
+            "name": "NoProviderError"
+          },
+          {
+            "type": 1,
+            "content": {
+              "json": "There was an error while adding the SAP NetWeaver provider. Please reach out to support for further assistance.",
+              "style": "warning"
+            },
+            "conditionalVisibilities": [
+              {
+                "parameterName": "param_check_provider_error",
+                "comparison": "isEqualTo",
+                "value": "-1"
+              },
+              {
+                "parameterName": "param_check_error",
+                "comparison": "isEqualTo",
+                "value": "0"
+              }
+            ],
+            "name": "ProviderFailure"
+          },
+          {
+            "type": 1,
+            "content": {
+              "json": "It may take up to 15 minutes for the initial telemetry information to be available for review. Please check again later.",
+              "style": "warning"
+            },
+            "conditionalVisibility": {
+              "parameterName": "param_check_error",
+              "comparison": "isEqualTo",
+              "value": "2"
+            },
+            "name": "IngestionError"
+          },
+          {
+            "type": 1,
+            "content": {
+              "json": "We have a newer version of SAP monitor. If you would like to see more metrics, please upgrade by deleting and creating a new SAP monitor.",
```

Comment:
Why would they need to delete and deploy completely new monitor? We upgrade them automatically